.github/workflows/terraform.yaml

```diff
@@ -137,16 +142,37 @@ jobs:
           path: /home/runner/.tflint.d/plugins
           key: ${{ runner.os }}-tflint-${{ hashFiles('.tflint.hcl') }}
 
-      - name: Run tflint with review comment on PR
-        uses: reviewdog/action-tflint@master
+      - name: Setup TFLint
+        uses: terraform-linters/setup-tflint@v2
         with:
+          tflint_version: latest
```

Comment:
Let's pin to specific version rather than latest.

Reply:
pinned